Automattic/jetpack
# Docker: Clean up Apache pid file on start

Sometimes WordPress Apache gets messed up which ends up in the PID file not being cleaned up. Obviously, it would not allow starting the Apache server once again. 

Original WordPress Docker image takes advantage of `apache2-foreground` script (https://github.com/docker-library/php/blob/master/apache2-foreground) which basically does the same as this PR.   

#### Jetpack product discussion


n/a
#### Does this pull request change what data or activity we track or use?


n/a
#### Testing instructions:






* Apply patch
* build Docker image: `jetpack docker build-image`
* Start up Docker as usual. 
* It should work as expected
